test/second_q/circuit/library/test_bogoliubov_transform.py

```diff
@@ -49,7 +49,7 @@ def test_bogoliubov_transform(self, n_orbitals, num_conserving):
             orbital_energies,
             transformed_constant,
         ) = hamiltonian.diagonalizing_bogoliubov_transform()
-        matrix = converter.map(hamiltonian.second_q_op()).to_matrix()
+        matrix = converter._map(hamiltonian.second_q_op()).to_matrix()
```

Comment:
We should not call a private method here

Reply:
convert_only can be used as replacement here.